### PR TITLE
implement sponsors section with react marquee slider

### PR DESCRIPTION
**Pull request by RyanL123**
making this pr to check lighthouse stats and see netlify preview

### CODE REVIEW

File: src/components/Sponsors.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        >
+            {sponsors.map(sponsor => (
+                <Motion
+                    key={sponsor.id}
+                    initDeg={randomIntFromInterval(0, 360)}
+                    direction={
+                        Math.random() > 0.5 ? "clockwise" : "counterclockwise"
+                    }
+                    velocity={10}
+                    radius={100}
```
Comment (err53) [inline src/components/Sponsors.js]: maybe decrease the radius so it has to calculate less?

Comment (err53) [reply src/components/Sponsors.js]: maybe also increase the width of the box so the sponsors can spread out a bit more or smth idk

**Comment by err53** [comment]
the logos are so jittery wtf

**Comment by davidli3100** [comment]
Sponsor logos are gigantic - can we scale them down a bit?

Also, images in the carousel should be centered in the object-fit/crop

**Pull request merged**